tests/data/2

```diff
@@ -0,0 +1,14 @@
+# This is private data. Do not parse.
```

Comment:
lol "Do not parse"

Reply:
😆  I will remove it. Just copied the system file.

Reply:
not required, I'm just amused that we're parsing the data we're told not to parse.  Is there a better interface to query these details from networkd?

Reply:
I am not sure there is. But let me get back to you on this.

Reply:
After research, I see there is no proper way of getting the lease file details especially OPTION_245.

Reply:
Thanks for looking!